### PR TITLE
fix: #211 UI Redesign - Desktop - Settings

### LINKED ISSUE #211: UI Redesign - Desktop - Settings
**Description**
The task is to implement the new design for the settings menu in the desktop app.

**Assets and Design Screens**
All assets and design screens have already been created.

- The Figma design can be found here https://www.figma.com/file/gnhrfcg8OYvAz0zJXZz2r3/Atmosphere-Pro?node-id=1%3A2
- The Assets required can be found here https://www.figma.com/file/gnhrfcg8OYvAz0zJXZz2r3/Atmosphere-Pro?node-id=0%3A1

For help with using Figma please watch this short tutorial video
https://user-images.githubusercontent.com/69275762/192322842-2ac40f88-c396-4a66-933b-3c8abae516c3.mov

**Additional context**
This work should be done on the UI_Redesign branch https://github.com/atsign-foundation/atmosphere_pro/tree/ui_redesign

### PR DESCRIPTION
**Pull request by Chandram-Dutta**
**- What I did**
Added new desktop background
Added Settings Tab in sidebar
Added Settings Screen 
Fixes #211 

**- How I did it**
Designed and Coded the settings screen for desktop and added it to navigation drawer following the convention of the app

**- How to verify it**
Click on the setting tab in the sidebar

**- Description for the changelog**
Added new Settings Screen

### CODE REVIEW

**Comment by Chandram-Dutta** [comment]
Hey @tinashe404 This fixes #241

**Comment by tinashe404** [comment]
@Chandram-Dutta it looks like you worked on a ticket that wasn't assigned to you. This task is currently assigned to @AdiAr11

File: lib/desktop_screens/desktop_welcome_screen/desktop_welcome_screen.dart [hunk 1]
```diff
@@ -243,6 +241,7 @@ class _DesktopWelcomeScreenState extends State<DesktopWelcomeScreen> {
     TextStrings().sidebarTermsAndConditions,
     TextStrings().sidebarFaqs,
     TextStrings().sidebarContactUs,
+    TextStrings().sidebarSettings,
```
Comment (sachins-geekyants) [inline lib/desktop_screens/desktop_welcome_screen/desktop_welcome_screen.dart:244]: Is `sidebarSettings` variable added in `text_strings.dart` file ?

File: lib/desktop_screens/desktop_settings/desktop_settings.dart [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+                              path: 'atmospherepro@atsign.com'));
+                        },
+                        image: optionIcons[3],
+                      ),
+                      SizedBox(
+                        height: 12,
+                      ),
+                      SizedBox(
+                        height: 12,
+                      ),
```
Comment (sachins-geekyants) [inline lib/desktop_screens/desktop_settings/desktop_settings.dart:151]: Can we combine two sizedBoxes in one and give it a height of 14 ?

**Comment by Chandram-Dutta** [comment]
> @Chandram-Dutta This looks great! Only one suggestion! could you please resize the delete atsign modal so that it's around the same width as the backup keys one?
> Screenshot 2022-10-25 at 11 33 44 AM
> Screenshot 2022-10-25 at 11 33 52 AM
> 
> @tinashe404 will do it...

**Comment by Chandram-Dutta** [comment]
> @Chandram-Dutta it looks like you worked on a ticket that wasn't assigned to you. This task is currently assigned to @AdiAr11 

Yes, i did the mobile version and there were no followups on issue... So i used the codebase of the mobile to desktop...

**Comment by tinashe404** [comment]
@Chandram-Dutta we accepted and merged your mobile PR for settings

**Pull request closed without merging**